src/v2/schema/application.js

```diff
@@ -23,7 +23,6 @@ export const typeDef = gql`
 
     name: String
     namespace: String
-    selfLink: String
```

Comment:
@KevinFCormier Looks like the App queries are still requesting `selfLink`.  https://github.com/open-cluster-management/application-ui/blob/73a6302cc821b5726cee1989f52b3f5752a2cd2b/lib/client/queries.js

@SherinV If we remove the field, it will break the client queries.  The missing field will cause the entire query to fail.  Let's first deprecate this field and we'll remove it later once the App ui get updated.  In the meantime we can just return an empty string.